AuroraStudiosDigipen/Borealis
# Added fbx loader and font loader to editor

Test case:

-FBX loading
    - add mesh filter component
    - open meshes folder and drag any of the model to the mesh button
    - *the meshes are huge so you'll need to scroll out to see the full view
    -*right now they are not using any textures so they will use the one currently binded

-Text
    -add text component
    -type text into text box


Things added :
Loading of fbx files from editor
Loading of fonts from editor
Added text component

Things to add:
-FBX
     -Texture uv mapping
     -materials
-Text
     -fix fonts scaling issues and rough edges
     -font size
     -font color
